### ISSUE TITLE
Text readability issue in light mode on Home page

### ISSUE THREAD

**Issue by parnikachoudhary** [sayeeg-11/Pixel_Phantoms#309]
### Describe the Bug

In light mode, the "LOADING EVENT" and "BROWSE ALL MODULES" text on the Home page are not clearly readable due to low color contrast between the text and the background. The text appears too light, making it difficult for users to read the content properly in light mode.

This negatively impacts user experience and accessibility.

### Steps to Reproduce

1. Open the Pixel Phantoms website:
https://sayeeg-11.github.io/Pixel_Phantoms/index.html

2. Switch to light mode.

3. Scroll to the Home page and observe the text for the "LOADING EVENT" and “BROWSE ALL MODULES".

### Expected Behavior

Text on the Home page, including the "LOADING EVENT" and "BROWSE ALL MODULES", should have sufficient color contrast and be clearly readable against the background in light and dark modes.

### Additional Context

##Screenshots 📸 

<img width="1855" height="902" alt="Image" src="https://github.com/user-attachments/assets/e274661a-1bca-4ead-8d07-6f13cbed649f" />

<img width="1876" height="955" alt="Image" src="https://github.com/user-attachments/assets/85e43c97-ef30-4203-bd4f-60ff0e5604ed" />
